### PR TITLE
context: uniform the import naming of context, part 1

### PR DESCRIPTION
**Pull request by leoppro**
### What problem does this PR solve? 
all context of std lib should be named as `context`
all context of cdc package should be named as `cdcContext`

this pr modified the code in `ticdc/cdc/processor/pipeline` package.

### Check List 

Tests 

 - Unit test

### Release note

- No release note

### CODE REVIEW

File: cdc/processor/pipeline/sink.go [hunk 1]
```diff
@@ -53,11 +53,13 @@ func (s TableStatus) String() string {
...
+// Load TableStatus with THREAD-SAFE
+func (s *TableStatus) Load() TableStatus {
 	return TableStatus(atomic.LoadInt32((*int32)(s)))
 }
 
-func (s *TableStatus) store(new TableStatus) {
+// Store TableStatus with THREAD-SAFE
+func (s *TableStatus) Store(new TableStatus) {
 	atomic.StoreInt32((*int32)(s), int32(new))
 }
```
Comment (amyangfei) [inline cdc/processor/pipeline/sink.go:64]: Will `Load` and `Store` be used by other packages?

Comment (leoppro) [reply cdc/processor/pipeline/sink.go:64]: yes, in new owner

**Pull request merged**